worker/codalabworker/local_run/local_run_state.py

```diff
@@ -71,6 +73,7 @@ class LocalRunStage(object):
         'bundle_path',
         'resources',
         'start_time',
+        'container',
```

Comment:
TODO: Make sure at all locations this is initialized, container is set to `None`